src/main/java/uk/gov/hmcts/divorce/common/notification/Applicant1AppliedForFinalOrderNotification.java

```diff
@@ -94,6 +108,21 @@ public void sendToApplicant1Solicitor(CaseData caseData, Long caseId) {
                 ENGLISH,
                 caseId
             );
+        } else {
+            if (isEmpty(caseData.getApplicant1().getSolicitor().getEmail())) {
```

Comment:
Could this get called with no solicitor? Only then would get null pointer I think

Reply:
i guess it won't tho cos this is the solicitor method